paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/ClusterControllerTest.java

```diff
@@ -100,10 +100,13 @@ public void testGetCluster() throws Exception {
     @Test
     @Order(3)
     public void testListClusters() throws Exception {
+        ClusterInfo cluster = new ClusterInfo();
```

Comment:
I don't think this test case has verified above changes.

Reply:
Thanks for the review. The previous mapper xml was wrong. There was no <where></where> tag, so if you add filter conditions when disconnecting the interface, an error will be reported. This pr only added the <where></where> tag, test test. A filter condition of type Flink was passed, which is currently working.

Reply:
Thanks for the review. The previous mapper xml was wrong. There was no <where></where> tag, so if you add filter conditions when disconnecting the interface, an error will be reported. This pr only added the <where></where> tag, test test. A filter condition of type Flink was passed, which is currently working.

Reply:
Thanks for the review. The previous mapper xml was wrong. There was no where tag, so if you add filter conditions when disconnecting the interface, an error will be reported. This pr only added the where tag, test test. A filter condition of type Flink was passed, which is currently working.

Reply:
Why not this test case verify that the previous mapper xml was wrong?